drone:
  max_velocity: 0.5
  min_velocity: -0.5
  safety_distance: 0.5
  collision_threshold: 0.4
  sensor_data_dimension:
    depth_camera:
      width: 64
      height: 48
    rplidar: 360
goal:
  reach_threshold: 0.5
reward_criteria:
  dense:
    coeff:
      lambda_p: 1
      lambda_w: 0.0015915494309189533 # 1/(200*pi)
      lambda_g: 0.15
      epsilon: 0.05
    no_motion: -0.05
    reach: 1
    collision: 0
  sparse:
    reach: 1
    collision: 0
